arrows/ocv/image_container.cxx

```diff
@@ -192,48 +197,63 @@ ::vital_to_ocv(const vital::image& img, image_container::ColorMode cm)
       out.addref();
     }
     // TODO use MatAllocator to share memory with image_memory
-    if(cm == RGB || out.channels() == 1 )
+    if( cm == RGB || cm == OTHER_COLOR_MODE || out.channels() < 3 || out.channels() > 4 )
```

Comment:
It might be better to use `cm != BGR` instead of `cm == RGB || cm == OTHER_COLOR_MODE` both because it is more compact and also because the latter will break if more color modes are added in the future.